src/Tizen.TV.UIControls.Forms/Renderer/GridViewRenderer.cs

```diff
@@ -0,0 +1,246 @@
+                Control.ItemAlignmentX = Element.ItemHorizontalAlignment;
+                Control.ItemAlignmentY = Element.ItemVerticalAlignment;
+                Control.Style = Element.ThemeStyle;
+                Control.ItemSelected += OnItemSelected;
+                Control.ItemRealized += OnItemRealized;
+
+                SmartEvent<GenGridItemEventArgs> _focused = new SmartEvent<GenGridItemEventArgs>(Control, Control.RealHandle, "item,focused", OnSmartEvent);
+                SmartEvent<GenGridItemEventArgs> _unfocused = new SmartEvent<GenGridItemEventArgs>(Control, Control.RealHandle, "item,unfocused", OnSmartEvent);
+                _focused.On += OnItemFocused;
+                _unfocused.On += OnItemUnfocused;
+
+                if (Element.ItemsSource != null)
+                {
+                    UpdateItemsSource();
+                }
+            }
+            base.OnElementChanged(e);
+        }
+
+        protected override void OnElementPropertyChanged(object sender, PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName == GridView.ItemWidthProperty.PropertyName)
+            {
+                Control.ItemWidth = Element.ItemWidth;
+            }
+            else if (e.PropertyName == GridView.ItemHeightProperty.PropertyName)
+            {
+                Control.ItemHeight = Element.ItemHeight;
+            }
+            else if (e.PropertyName == GridView.ItemVerticalAlignmentProperty.PropertyName)
+            {
+                Control.ItemAlignmentX = Element.ItemVerticalAlignment;
+            }
+            else if (e.PropertyName == GridView.ItemHorizontalAlignmentProperty.PropertyName)
+            {
+                Control.ItemAlignmentY = Element.ItemHorizontalAlignment;
+            }
+            else if (e.PropertyName == GridView.HorizontalScrollBarVisibleProperty.PropertyName)
+            {
+                Control.HorizontalScrollBarVisiblePolicy =
+                    Element.HorizontalScrollBarVisible ? ElmSharp.ScrollBarVisiblePolicy.Visible : ScrollBarVisiblePolicy.Invisible;
+            }
+            else if (e.PropertyName == GridView.VerticalScrollBarVisibleProperty.PropertyName)
+            {
+                Control.VerticalScrollBarVisiblePolicy =
+                    Element.VerticalScrollBarVisible ? ElmSharp.ScrollBarVisiblePolicy.Visible : ScrollBarVisiblePolicy.Invisible;
+            }
+            else if (e.PropertyName == GridView.OrientationProperty.PropertyName)
+            {
+                Control.IsHorizontal = Element.Orientation == GridViewOrientation.Horizontal ? true : false;
+            }
+            else if (e.PropertyName == GridView.ItemsSourceProperty.PropertyName)
+            {
+                UpdateItemsSource();
+            }
+            else if (e.PropertyName == GridView.SelectedItemProperty.PropertyName)
+            {
+                UpdateSelectedItem();
+            }
+            base.OnElementPropertyChanged(sender, e);
```

Comment:
Changes of `ItemTemplate` is not handled

Reply:
I fixed it. Thank you.